.circleci/generate.sh

```diff
@@ -332,8 +336,72 @@ delete_repeated_jobs()
   fi
 }
 
+# Define function to leave only a single config run for each test group
+# The first and only argument is the file name.
+dedup_jobs()
+{
+  delete_job "$1" "j11_cqlsh_dtests_py311_offheap"
```

Comment:
I have a small preference for replacing all these deletions of particular jobs with something that would delete everything except particular jobs the other way around. Otherwise, I can see how people will start adding new jobs to the workflows without thinking whether they should be in the skinny workflow. Also, the list will be smaller and less prone to merge conflicts, etc. However, I leave it to you, @bereng, to decide whether you want to change this. Not a blocker here.

Reply:
If you look at the script details we unfortunately don't support that currently as we don't use any yaml processing tooling but rely on regexps.

Reply:
Not a blocker, it was just a suggestion, resolving